pkg/eval/builtin_fn_styled.go

```diff
@@ -68,7 +68,12 @@ var errStyledSegmentArgType = errors.New("argument to styled-segment must be a s
 //     -   Any color from the xterm 256-color palette, as `colorX` (such as
 //    `color12`).
 //
-//     -   A 24-bit RGB color, as `#RRGGBB`, such as `#778899`.
+//     -   A 24-bit RGB color written as `#RRGGBB` such as `'#778899'`. Note
```

Comment:
Please put notes in a separate paragraph starting with `NOTE:`. This makes it clear that the information is non-normative.

Reply:
Done